cmd/scollector/collectors/procstats_linux.go

```diff
@@ -134,6 +142,12 @@ func c_procstats_linux() (opentsdb.MultiDataPoint, error) {
 					tags["cpu"] = tag_cpu
 				}
 				Add(&md, "linux.cpu"+metric_percpu, value, tags, metadata.Counter, metadata.CHz, cpu_stat_desc[CPU_FIELDS[i]])
+
+				vf, err := strconv.ParseFloat(value, 64)
+				if err != nil {
+					return nil
+				}
+				Add(&md, "linux.percent_cpu"+metric_percpu, vf/total_cpu*100.0, tags, metadata.Counter, metadata.Pct, cpu_stat_desc[CPU_FIELDS[i]])
```

Comment:
So, with OpenTSDB, if I was to query a single tag value, it would convert that to a percentage of the sum of all tag values somehow?  I'm using InfluxDB.